Cargo.toml

```diff
@@ -11,17 +11,20 @@ repository = "https://github.com/maidsafe/self_encryption"
 version = "0.16.0"
 
 [dependencies]
+async-trait = "~0.1.30"
 aes = "0.3.2"
 block-modes = "0.3.3"
 bincode = "~1.1.4"
 brotli = "~3.3.0"
-futures = "~0.1.28"
+criterion = "~0.3"
```

Comment:
Seems like this is a dependency required for benching. This can go in the dev-dependencies.